src/main/java/com/laytonsmith/abstraction/events/MCPrepareAnvilEvent.java

```diff
@@ -0,0 +1,16 @@
+package com.laytonsmith.abstraction.events;
+
+import com.laytonsmith.abstraction.MCHumanEntity;
+import com.laytonsmith.abstraction.MCInventory;
+import com.laytonsmith.abstraction.MCItemStack;
+
+import java.util.List;
+
+public interface MCPrepareAnvilEvent extends MCInventoryEvent {
+	@Override
```

Comment:
What is this override doing here?

Reply:
Uh, nothing. Actually, that entire `getInventory()` definition doesn't need to exist. It's defined in `MCInventoryEvent`. Removed it.